pyclient/pydeephaven/table_interface.py

```diff
@@ -328,72 +312,68 @@ def raj(self, table: Any, keys: List[str], columns_to_add: List[str] = [], match
 
         Raises:
             DHError
-
         """
         match_rule = MatchRule.GREATER_THAN if match_rule == MatchRule.GREATER_THAN else MatchRule.GREATER_THAN_EQUAL
-        table_op = AsOfJoinOp(table=table, keys=keys, columns_to_add=columns_to_add,
+        table_op = AsOfJoinOp(table=table, keys=on, columns_to_add=joins,
                               match_rule=match_rule)
         return self.table_op_handler(table_op)
 
-    def head_by(self, num_rows: int, column_names: List[str]):
+    def head_by(self, num_rows: int, by: List[str]):
         """ Perform a head-by operation on the table and return the result table.
```

Comment:
```suggestion
        """ Perform a head-by aggregation on the table and return the result table.
```
This is what the latter docs do.